portal/views.py

```diff
@@ -0,0 +1,192 @@
+from django.db.models import Count, Q
+from django.db.models.functions import Trunc
+from django.utils import timezone
+from rest_framework import generics, viewsets
+from rest_framework.decorators import action
+from rest_framework.permissions import IsAdminUser, IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from portal.logic import get_demographic_breakdown, get_user_populations
+from portal.models import Poll, PollOption, PollVote, TargetPopulation
+from portal.permissions import OwnerPermission, TimeSeriesPermission
+from portal.serializers import (
+    PollOptionSerializer,
+    PollSerializer,
+    PollVoteSerializer,
+    RetrievePollSerializer,
+    RetrievePollVoteSerializer,
+    TargetPopulationSerializer,
+)
+
+
+class TargetPopulations(generics.ListAPIView):
+    """List view to see which populations a poll can select"""
+
+    permission_classes = [IsAuthenticated]
+    serializer_class = TargetPopulationSerializer
+    queryset = TargetPopulation.objects.all()
+
+
+class Polls(viewsets.ModelViewSet):
+    """
+    browse:
+    returns a list of Polls that are valid and
+    haven't been filled out by user
+
+    review:
+    returns a list of unapproved Polls for admin
+
+    create:
+    Create a Poll
+
+    partial_update:
+    Update certain fields in the Poll.
+    Only user who creates Poll can edit it
+
+    destroy:
+    Delete a Poll.
+    """
+
+    permission_classes = [OwnerPermission]
+    queryset = Poll.objects.all()
```

Comment:
I'm not sure, but I think you don't need to/shouldn't define this if you also use `get_queryset`